src/include/traffic_cop/traffic_cop.h

```diff
@@ -114,10 +114,17 @@ class TrafficCop {
   executor::ExecuteResult p_status_;
 
   bool is_queuing_;
+  
+  inline void SetDefaultDatabaseName(std::string default_database_name) {
+    default_database_name_ = default_database_name;
+  }
 
 //  struct event* event_;
  private:
 
+  // Default database name
+  std::string default_database_name_ = DEFAULT_DB_NAME;
```

Comment:
Why do we need to duplicate this here and `AbstractOptimizer`? We should set it in one location so that it will be easier for us to replace this later.

Reply:
I have removed the one in AbstractOptimizer.